best-practices/README.md

```diff
@@ -189,10 +189,9 @@ Email
 
 JavaScript
 ----------
-* Use Coffeescript, ES6 with [babel], or another language that compiles to
-  JavaScript
+* Use the latest stable JavaScript syntax with a transpiler, such as [babel].
```

Comment:
ES6 (as it stands) is [not completely supported by all browsers](https://kangax.github.io/compat-table/es6/), so you need a transpiler to use it.

If you had to use ES5, that'd be fine -- you could still use a transpiler and use ES6 syntax, which gets compiled to ES5 syntax.

In the future, when ES7 comes out, then we will still need a transpiler to use it, as not all browsers will support that, either.

So, I think we will need a transpiler either way.